test/src/test/java/hudson/tasks/ArtifactArchiverTest.java

```diff
@@ -204,12 +209,12 @@ public void configRoundTrip() throws Exception {
         aa = j.configRoundtrip(aa);
         assertEquals("*.txt", aa.getArtifacts());
         assertNull(Util.fixEmpty(aa.getExcludes()));
-        assertEquals("{artifacts=*.txt}", DescribableModel.uninstantiate_(aa).toString());
+        assertEquals("{artifacts=*.txt, basePath=}", DescribableModel.uninstantiate_(aa).toString());
```

Comment:
Nope, this means you messed up. [Guide](https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#handling-default-values)